lib/init.coffee

```diff
@@ -12,6 +12,10 @@ module.exports =
       default: ['Default', 'Dollar', 'Generalise']
       items:
         type: 'string'
+    hlintIgnoreReduceDuplication:
```

Comment:
As this is already behind the `linter-hlint` tag, there is no need to prefix this with `hlint`. Can you rename this to `ignoreReduceDuplication`?